<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QDialog" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back55.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Downloads/physioform-app-desktop-en-c-et-Qt-designer-master/physioform-app-desktop-en-c-et-Qt-designer-master/photos/send.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>450</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#SendButton{
 background-color: #f6f6f6;
    border: 2px solid #31a4b4;
    border-radius: 10px;
    padding: 5px;
    selection-background-color: #31a4b4;
}

#SendButton:hover{
background-color: #5bab99;
}</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="chatDisplayTextEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#chatDisplayTextEdit{
    background-color: #f6f6f6;
    border: 2px solid #31a4b4;
    border-radius: 10px;
}

#chatDisplayTextEdit QHeaderView::section {
    background-color: #31a4b4;
    color: #ffffff;
    border: 1px solid #5bab99;
    border-radius: 5px;
}

#chatDisplayTextEdit QTableCornerButton::section {
    background-color: #31a4b4;
    border: 1px solid #5bab99;
}

#chatDisplayTextEdit QTableView {
    alternate-background-color: #e0e0e0;
}

#chatDisplayTextEdit QTableView:selected {
    background-color: #31a4b4;
    color: #ffffff;
}

#chatDisplayTextEdit QLineEdit {
    border: 1px solid #5bab99;
    border-radius: 5px;
    padding: 2px;
    selection-background-color: #31a4b4;
}

#chatDisplayTextEdit QTableView::indicator:checked {
    background-color: #31a4b4;
    border: 1px solid #5bab99;
}

#chatDisplayTextEdit QTableView::indicator:unchecked {
    background-color: #f6f6f6;
    border: 1px solid #5bab99;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#messageTextEdit {
    background-color: #f6f6f6;
    border: 2px solid #31a4b4;
    border-radius: 10px;
}

##messageTextEdit QHeaderView::section {
    background-color: #31a4b4;
    color: #ffffff;
    border: 1px solid #5bab99;
    border-radius: 5px;
}

##messageTextEdit QTableCornerButton::section {
    background-color: #31a4b4;
    border: 1px solid #5bab99;
}

##messageTextEdit QTableView {
    alternate-background-color: #e0e0e0;
}

##messageTextEdit QTableView:selected {
    background-color: #31a4b4;
    color: #ffffff;
}

##messageTextEdit QLineEdit {
    border: 1px solid #5bab99;
    border-radius: 5px;
    padding: 2px;
    selection-background-color: #31a4b4;
}

##messageTextEdit QTableView::indicator:checked {
    background-color: #31a4b4;
    border: 1px solid #5bab99;
}

##messageTextEdit QTableView::indicator:unchecked {
    background-color: #f6f6f6;
    border: 1px solid #5bab99;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
